.gitignore

```
@@ -0,0 +1 @@
+/node_modules
```

```suggestion
/node_modules
.DS_Store
```

This is a Mac OS file that you can ignore. To untrack it from git, you can run `git rm -r --cached .DS_Store && git commit -m "Untracks DS_Store"`